scheme.html

```diff
@@ -84,14 +81,19 @@ <h1 id="authority">Loading...</h1>
           <button type="button" id="delete" disabled>Delete</button>
         </div>
       </div>
-
       <div
         class="sidebar-toggle rounded-rect"
         onclick="document.getElementById('sidebar').classList.toggle('collapsed'); app.map.resize();"
       >
         &rarr;
       </div>
-    </aside>
+      <div class="sidebar-content content-container">
```

Comment:
I don't think we want this <div> again. The two items should be at the same "level" they were both, just moved down